ChatWithAuthSample/AuthorizationSample/Controllers/ValuesController.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
```

Comment:
Why leave this file in if unused anywhere?

Reply:
Remnant that I can definitely delete.